ci/ci.sh

```diff
@@ -389,6 +389,8 @@ install_ray() {
 }
 
 validate_wheels_commit_str() {
+  echo "TODO: validate_wheels_commit_str disabled"
```

Comment:
Let's remove the `"${folder}.data/purelib/ray/__init__.py"` line below at line 418 which is causing issues actually.

Reply:
If I remove that line, isn't the function just checking if BUILDKITE_COMMIT/TRAVIS_COMMIT is set? Shouldn't we just remove the whole check?